src/core.cpp

```diff
@@ -692,4 +682,47 @@ double Core::undefinedIndex() {
     return -1;
 }
 
+std::vector<Core *> Core::core_registry;
+
+/*! Create a core and add it to the registry 
+ */
+int Core::mkcore(bool logtofile, Logger::LogLevel loglvl, bool logtoscrn)
+{
```

Comment:
Not a big deal, I guess, but this syntax (brace on newline) doesn't match anything else in Hector.

Reply:
Weird.  Opening brace on its own line for function definitions (and nowhere else) goes all the way back to K&R.  Putting the opening brace on the definition line seems to be a Java thing.  K&R, Stroustrup, Linux foundation, and GNU all put the opening brace of a function definition on its own line.

Examples:
https://www.kernel.org/doc/html/v4.10/process/coding-style.html#placing-braces-and-spaces
https://en.wikipedia.org/wiki/Indentation_style#K&R_style
https://en.wikipedia.org/wiki/Indentation_style#GNU_style

Personally, I prefer the Stroustrup formatting (as defined in _The C++ Programming Language_ -- it turns out Stroustrup himself now does something different), and I _really_ hate cuddling the brace on a function definition.  It messes with some text editors' functions for jumping to the beginning of a function, and it makes it harder for me, personally, to visually scan for the beginning of a function.

Again, I'm not saying we should go back and change all the existing code, but we should consider encouraging the Stroustrup formatting style going forward.